frontend/src/components/app/app.tsx

```diff
@@ -12,6 +13,17 @@ const App: React.FC = () => {
   return (
     <>
       <div className="App">
+        <ToastContainer
```

Comment:
let's use another package `react-redux-toastr` and create a new common component for it